lib/sufia/configuration.rb

```diff
@@ -84,10 +84,14 @@ def audit_user_key
       @audit_user_key ||= 'audituser@example.com'
     end
 
-    # TODO: this is called working_path in curation_concerns
```

Comment:
I should think that it will be fine. I'll test it out. I only renamed it because I thought it would be a bit more descriptive as to what it was modifying. But I can definitely understand your concern now that you point it out.